### PR TITLE
MenuScreen

### PR DESCRIPTION
**Pull request by Xinoi**
Made a prototype of a Menu Screen. It looks bad and isnt done but it a start :).

### CODE REVIEW

File: core/src/de/jjj/spacegame/MenuScreen.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    private float startButtonWidth = 200;
+    private float startButtonHeigth = 100;
+    private float startButtonx = Gdx.graphics.getWidth() / 2 - startButtonWidth/2;
+    private float startButtony = Gdx.graphics.getHeight() - Gdx.graphics.getHeight() / 4 * 2;
+
+    private float settingsButtonWidth = 250;
+    private float settingsButtonHeigth = 100;
+    private float settingsButtonx = Gdx.graphics.getWidth() / 2 - settingsButtonWidth/2;
+    private float settingsButtony = Gdx.graphics.getHeight() - Gdx.graphics.getHeight() / 4 * 3;
+
```
Comment (Jan130) [inline core/src/de/jjj/spacegame/MenuScreen.java:33]: Maybe you should move the initialisation of variables to the constructor.

**Review by Jan130 — CHANGES_REQUESTED** [review]
Move variable initialisation in MenuScreen.java to the constructor

**Comment by itsSanoi** [comment]
Maby add a tital to the screen.
After that change and the change from Jan iḿ rady for a merge

**Comment by Xinoi** [comment]
fixed change request from jan

**Review by Jan130 — APPROVED** [review]

**Pull request merged**